src/components/procedure/_remove-ruleset.mdx

```diff
@@ -1,7 +1,7 @@
 You can disable a specific rule or ruleset to prevent Semgrep Code from using it when scanning your codebase.
 
 :::info
-- When you disable a rule, all previous findings generated by that rule are still retained.
+When you disable a rule, existing findings from that rule remains open until you re-scan your code.
```

Comment:
So - **disabling** a rule == **removing** the findings, and the findings total adjusts accordingly (and affects fix rate, I am guessing, because the total number of findings will change). If this is correct feel free to resolve this comment, I'll approve now.